trk/src/Screens/ClimbCreate/index.js

```diff
@@ -2,7 +2,7 @@ import React, { useState, useEffect, useContext } from "react";
 import DropDownPicker from 'react-native-dropdown-picker';
 // import ImagePicker from 'react-native-image-crop-picker';
 
-import { SafeAreaView, View, Text, StyleSheet, TextInput, Image, Button, Alert, TouchableOpacity, Platform, TouchableWithoutFeedback, Keyboard, ScrollView } from "react-native";
+import { SafeAreaView, View, Text, StyleSheet, TextInput, Image, Button, Alert, TouchableOpacity, Platform, TouchableWithoutFeedback, Keyboard, ScrollView, KeyboardAvoidingView } from "react-native";
```

Comment:
This makes the view scrollup when the keyboard is clicked to avoid hiding the input field